exercises/poker/.meta/src/reference/java/Card.java

```diff
@@ -16,10 +16,14 @@ int getSuit() {
     }
 
     private int parseRank(String card) {
-        return "..23456789TJQKA".indexOf(card.charAt(0));
+    	if (card.substring(0, 2).equals("10")) {
+			return 10;
+		}
+		String s = "..23456789TJQKA";
+		return s.indexOf(card.charAt(0));
```

Comment:
The indentation appears to be a bit off here. Please try to make it consistent with the rest of the file 🙂 

Also, I think the last two lines could be one line as before:
```
return "..23456789TJQKA".indexOf(card.charAt(0));
```

Or if you would rather extract the string as a variable, then it should be given a useful and descriptive name, e.g. `possibleRanks` 🙂